pages/release.md

```diff
@@ -1,15 +1,48 @@
 ---
 layout: page
-title: Releases
+title: Current Releases
```

Comment:
Since the page includes both current and archived releases, I think the previous "Releases" is better. Was this done to inject the title into the page? Or just the `<head><title /></head>`?

Reply:
I made the the title "Releases", switched to the default layout so title isn't injected into page and then created "Current" and "Archive" sections.